### ISSUE TITLE
In consistent behavior on the ? for glossary

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#634]
The glossary was not sliding out. Ask Rebecca about her experience she was encountering errors with chrome. I haven't been able to reproduce.

**Comment by rdpiazza** [comment]
Glossary is not sliding out when I click on words identified as having definitions or when I try to open the Glossary.  The following actions did not open the glossary:

* Clicking the Glossary icon in the upper right corner
* Clicking "Candidate (?)" above the "Enter a candidate" text box
* Clicking "Principal Campaign Committee (?)" at the bottom of the page to the left of the animated gif

I was able to open glossary items using Chrome yesterday.  I restarted Chrome to determine if there was an issue with my browser and continue to see the issue.
